### DISCUSSION TITLE
【抽卡展示】常驻五星只有玛卡巴卡，肿么办xdm，她是不是赖上我了

### DISCUSSION THREAD

**Discussion by guiguihha** [share121/inter-knot#578] [其他]
![Screenshot_2024-08-10-11-48-59-378_com tencent mm-edit](https://github.com/user-attachments/assets/d1f9a730-baf3-4581-843a-61668ed00214)

平民玩家，只充了6元礼包，11号已经2影了，结果防卫战第10层连s都拿不到，哭

**Comment by aotianyh** [comment]
你不许玩绝区零

**Comment by summer114514** [comment]
建议退坑

**Comment by dominic121-dot** [comment]
这号废了，建议退坑
